docs/reference/tools-reference/new-llm-vision-tool.md

```diff
@@ -0,0 +1,52 @@
+# LLM Vision
+
+## Introduction
+Prompt flow LLM vision tool enables you to leverage your AzureOpenAI GPT-4 Turbo or OpenAI's GPT-4 with vision to analyze images and provide textual responses to questions about them.
+
+## Prerequisites
+Create OpenAI or Azure OpenAI resources:
+
+- **OpenAI**
+
+    Sign up account [OpenAI website](https://openai.com/)
+
+    Login and [Find personal API key](https://platform.openai.com/account/api-keys)
+
+- **Azure OpenAI (AOAI)**
+
+    Create Azure OpenAI resources with [instruction](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal)
+
+    Browse to [Azure OpenAI Studio](https://oai.azure.com/) and sign in with the credentials associated with your Azure OpenAI resource. During or after the sign-in workflow, select the appropriate directory, Azure subscription, and Azure OpenAI resource.
+
+    Under Management select Deployments and Create a GPT-4 Turbo with Vision deployment by selecting model name: `gpt-4` and model version `vision-preview`.
+
+
+   
+## **Connections**
+
+Setup connections to provisioned resources in prompt flow.
+
+| Type        | Name     | API KEY  | API Type | API Version |
+|-------------|----------|----------|----------|-------------|
+| OpenAI      | Required | Required | -        | -           |
+| AzureOpenAI | Required | Required | Required | Required    |
+
+## Inputs
+
+| Name                    | Type        | Description                                                                                    | Required |
+|-------------------------|-------------|------------------------------------------------------------------------------------------------|----------|
+| connection              | AzureOpenAI | the AzureOpenAI connection to be used in the tool                                              | Yes      |
```

Comment:
Can also accept OpenAI type?

Reply:
removed here. Since connections are introduced under the connections heading/section